var/spack/repos/builtin/packages/yosys/package.py

```diff
@@ -9,25 +9,19 @@ class Yosys(MakefilePackage):
     """Yosys is a framework for RTL synthesis tools. It currently has extensive
     Verilog-2005 support and provides a basic set of synthesis algorithms for
     various application domains.
-
-    Yosys can be adapted to perform any synthesis job by combining the existing
-    passes (algorithms) using synthesis scripts and adding additional passes
-    as needed by extending the yosys C++ code base.
-
-    Yosys is free software licensed under the ISC license (a GPL compatible
-    license that is similar in terms to the MIT license or the 2-clause BSD license).
```

Comment:
Please describe why this part of the description was removed. The license paragraph at the end looks fine to remove, but you didn't put anything about this change in your PR description.